packages/protocol/contracts/common/FixidityLib.sol

```diff
@@ -214,6 +191,7 @@ library FixidityLib {
    * Test multiply(maxFixedMul()+1,maxFixedMul()+1) fails
    */
   function multiply(Fraction memory x, Fraction memory y) internal pure returns (Fraction memory) {
+    require(x.value <= maxFixedMul() && y.value <= maxFixedMul());
```

Comment:
I believe there are unit tests that verify this reverts anyways without this check when the inputs are too large. I understand being conservative, as this library aims to be safe, but it also aims to be efficient, do we need an additional `require` here?

Reply:
Good point. Will make the functions that expose these values external, so that clients that consume this code can do input sanitation more easily.

Reply:
Actually nevermind, as this increases bytecode size. Going to move everything to comments.